politeiawww/cmd/politeiawww_dbutil/politeiawww_dbutil.go

```diff
@@ -414,65 +430,123 @@ func cmdStubUsers() error {
 	return nil
 }
 
-func cmdMigrate() error {
-	// Connect to LevelDB
+func connectLevelDB() (user.Database, error) {
 	dbDir := filepath.Join(*dataDir, network)
 	_, err := os.Stat(dbDir)
 	if err != nil {
 		if os.IsNotExist(err) {
 			err = fmt.Errorf("leveldb dir not found: %v", dbDir)
 		}
-		return err
+		return nil, err
 	}
 
-	ldb, err := localdb.New(dbDir)
+	fmt.Printf("LevelDB     : %v\n", dbDir)
+	return localdb.New(dbDir)
+}
+
+func connectCockroachDB() (user.Database, error) {
+	err := validateCockroachParams()
 	if err != nil {
-		return err
+		return nil, fmt.Errorf("new cockroachdb: %v", err)
 	}
 
-	// Connect to CockroachDB
-	err = validateCockroachParams()
+	fmt.Printf(`CockroachDB : %v %v 
```

Comment:
This type of detailed printing should really only be done if a verbose (`-v`) flag is used.  Printing this by default creates a mess when you have scripts that use dbutil commands.  Here's the output for a script that I use to setup users.  It makes the output unreadable.

```
CockroachDB : localhost:26257 testnet3
                /home/luke/.cockroachdb/certs/clients/politeiawww/ca.crt
                /home/luke/.cockroachdb/certs/clients/politeiawww/client.politeiawww.crt
                /home/luke/.cockroachdb/certs/clients/politeiawww/client.politeiawww.key
                /home/luke/.politeiawww/sbox.key
User with username 'admin' admin status updated to true
CockroachDB : localhost:26257 testnet3
                /home/luke/.cockroachdb/certs/clients/politeiawww/ca.crt
                /home/luke/.cockroachdb/certs/clients/politeiawww/client.politeiawww.crt
                /home/luke/.cockroachdb/certs/clients/politeiawww/client.politeiawww.key
                /home/luke/.politeiawww/sbox.key
50 proposal credits added to account admin
CockroachDB : localhost:26257 testnet3
                /home/luke/.cockroachdb/certs/clients/politeiawww/ca.crt
                /home/luke/.cockroachdb/certs/clients/politeiawww/client.politeiawww.crt
                /home/luke/.cockroachdb/certs/clients/politeiawww/client.politeiawww.key
                /home/luke/.politeiawww/sbox.key
CockroachDB : localhost:26257 testnet3
                /home/luke/.cockroachdb/certs/clients/politeiawww/ca.crt
                /home/luke/.cockroachdb/certs/clients/politeiawww/client.politeiawww.crt
                /home/luke/.cockroachdb/certs/clients/politeiawww/client.politeiawww.key
                /home/luke/.politeiawww/sbox.key
User with username 'admin' admin status updated to true
CockroachDB : localhost:26257 testnet3
                /home/luke/.cockroachdb/certs/clients/politeiawww/ca.crt
                /home/luke/.cockroachdb/certs/clients/politeiawww/client.politeiawww.crt
                /home/luke/.cockroachdb/certs/clients/politeiawww/client.politeiawww.key
                /home/luke/.politeiawww/sbox.key
50 proposal credits added to account admin
CockroachDB : localhost:26257 testnet3
                /home/luke/.cockroachdb/certs/clients/politeiawww/ca.crt
                /home/luke/.cockroachdb/certs/clients/politeiawww/client.politeiawww.crt
                /home/luke/.cockroachdb/certs/clients/politeiawww/client.politeiawww.key
                /home/luke/.politeiawww/sbox.key
50 proposal credits added to account user
```